f2/src/utils/yupSchema.js

```diff
@@ -17,10 +22,16 @@ export const yupSchema = () => {
       message: <Trans id="locationInfoForm.Warning" />,
     }),
 
+    phoneExtensionSchema: Yup.string().matches(regexDef().phoneExtensionRegex, {
+      message: <Trans id="contactinfoForm.email.warning" />,
```

Comment:
Where is phoneExtensionRegex defined?

Reply:
Sorry  forget including/adding regex file, will add soon